resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/ServerResponseWriter.java

```diff
@@ -78,7 +78,8 @@ public static void writeNomapResponse(BuiltResponse jaxrsResponse, final HttpReq
       
       executeFilters(jaxrsResponse, request, response, providerFactory, method);
 
-      if (jaxrsResponse.getEntity() == null)
+      //[RESTEASY-1627] check on response.getOutputStream() to avoid resteasy-netty4 trying building a chunked response body for HEAD requests 
+      if (jaxrsResponse.getEntity() == null || response.getOutputStream() == null)

```

Comment:
Maybe I was being overly optimistic about being able to solve this problem in resteasy-jaxrs. It **feels** like it should be possible, but that and 10,000 lire will get you an uber ride. ;-)

So, +1.